pkg/virt-controller/services/template.go

```diff
@@ -87,6 +87,8 @@ const MULTUS_DEFAULT_NETWORK_CNI_ANNOTATION = "v1.multus-cni.io/default-network"
 // Istio list of virtual interfaces whose inbound traffic (from VM) will be treated as outbound traffic in envoy
 const ISTIO_KUBEVIRT_ANNOTATION = "traffic.sidecar.istio.io/kubevirtInterfaces"
 
+const ISTIO_INJECT_ANNOTATION = "sidecar.istio.io/inject"
```

Comment:
I don't like the fact that we're duplicating this from https://github.com/kubevirt/kubevirt/blob/37ec453d972a7b4957fb6c1160cfdba71d007512/pkg/virt-launcher/virtwrap/network/podinterface.go#L60.

Would you consider defining this in `.../pkg/network/...` or a network types / constants package under that tree ? 

The PR can go in as is, I'd just like to establish some ground rules on how to deal w/ this for the future.

Reply:
Done in the [1. commit ](https://github.com/kubevirt/kubevirt/pull/5797/commits/356690e9eb7d7c55f6593783358c991d9d7f95f2)
Added [2. commit](https://github.com/kubevirt/kubevirt/pull/5797/commits/588c3d10bb4eb751ab93824cf0f9a0293b1acdf4) where I use the defined constant in other places